docs/source/manual/openfpga_flow/run_fpga_task.rst

```diff
@@ -94,7 +94,7 @@ Declaring all the above sections are mandatory.
 General Section
 ^^^^^^^^^^^^^^^
 
-.. option:: fpga_flow=<yosys_vpr|vpr_blif>
+.. option:: fpga_flow=<yosys_vpr|vpr_blif|yosys>
```

Comment:
We should add explanation here about what each flow does. 
@ganeshgore What do you think?

Reply:
Yes, that will help. With increasing number of flows it is getting confusing. 
yosys_vpr  :  Verilog -> Blif -> VPR 
vpr_blif  :  Blif -> VPR